### ISSUE TITLE
finance.yahoo.com: nuisance

### ISSUE THREAD

**Issue by Alonzoxxx** [uBlockOrigin/uAssets#16555]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://finance.yahoo.com/quote/TLRY?p=TLRY&.tsrc=fin-srch`

### Category

nuisance

### Description

news/yahoo.com - finance/yahoo.com
modals, overlays disable browser function.

### Screenshot(s)

https://finance.yahoo.com/quote/TLRY?p=TLRY&.tsrc=fin-srch

### Configuration

<details>

```yaml
uBlock Origin: 1.46.0
Firefox: 109
filterset (summary): 
  network: 168336
  cosmetic: 171305
  scriptlet: 37928
  html: 1005
listset (total-discarded, last-updated): 
  added: 
    https://filters.adtidy.org/extension/ublock/filters/3.txt: 35850-14272, 8h.34m
    adguard-generic: 67965-6107, 8h.41m
    adguard-mobile: 8758-210, 7h.50m
    adguard-spyware-url: 897-4, 8h.39m
    block-lan: 44-0, 13d.1h.20m
    curben-phishing: 22754-3, 3h.38m
    curben-pup: 196-0, 3h.28m
    adguard-annoyance: 63004-5194, 8h.37m
    adguard-social: 19794-1691, 8h.36m
    fanboy-thirdparty_social: 69-1, 3d.21h.5m
    fanboy-annoyance: 86754-74807, 3d.21h.4m
    fanboy-cookiemonster: 45189-17433, 3d.21h.3m
    fanboy-social: 23505-13747, 5d.22h.7m
    ublock-annoyances: 4764-8, 8h.35m
    dpollock-0: 11397-472, 1d.22h.54m
  default: 
    user-filters: 35-0, never
    ublock-filters: 32555-311, 8h.47m
    ublock-badware: 4200-5, 8h.46m
    ublock-privacy: 307-1, 8h.45m
    ublock-abuse: 77-0, 8h.44m
    ublock-unbreak: 1879-3, 8h.43m
    ublock-quick-fixes: 524-0, 3h.40m
    easylist: 61117-17764, 23h.9m
    easyprivacy: 30886-1840, 23h.8m
    urlhaus-1: 6602-0, 3h.39m
    plowe-0: 3667-2, 13d.1h.20m
filterset (user): [array of 35 redacted]
trustedset: 
  added: [array of 2 redacted]
  removed: 
    about-scheme
switchRuleset: 
  added: [array of 3 redacted]
hostRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: 
  advancedUserEnabled: true
  ignoreGenericCosmeticFilters: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1633 ms
  maxAssetCacheWait: 1087 ms
popupPanel: 
  blocked: 65
  no-popups: true
  no-remote-fonts: true
  blockedDetails: 
    yahoo.com: 30
    oath.com: 4
    scorecardresearch.com: 1
    yimg.com: 30
```
</details>

**Comment by stephenhawk8054** [comment]
Do you mean this overlay?

Screenshot  
<details>

![image](https://user-images.githubusercontent.com/66517106/215372717-5ae4f791-0be2-45ae-829e-9a4a1f82595a.png)

</details>

**Comment by Alonzoxxx** [comment]
Hi, 
Yes, and a few others on yahoo's main pages.
Also, soap2day.com
Has pop up redirects.
I'm not too savvy, so I appreciate your help
Thanks.

**Comment by stephenhawk8054** [comment]
@ryanbr For `Fanboy's Annoyance`

**Comment by ryanbr** [comment]
Does that work?
```
finance.yahoo.com##.YDC-Overlay
finance.yahoo.com##html:style(overflow: auto !important; position: initial !important;)
```

**Comment by stephenhawk8054** [comment]
It works for me

**Comment by Alonzoxxx** [comment]
Thank you, Both!


-----Original Message-----
From: stephenhawk8054 ***@***.***>
Reply-To: uBlockOrigin/uAssets
***@***.***>
To: uBlockOrigin/uAssets ***@***.***>
Cc: Alonzoxxx ***@***.***>, Author ***@***.***>
Subject: Re: [uBlockOrigin/uAssets] finance.yahoo.com: nuisance (Issue
#16555)
Date: Sun, 29 Jan 2023 21:47:57 -0800

Closed #16555 as completed.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID:
***@***.***>
